enterprise/models.py

```diff
@@ -68,6 +68,8 @@ class Meta(object):
     active = models.BooleanField(default=True)
     history = HistoricalRecords()
 
+    identity_provider = models.PositiveIntegerField(null=True, blank=True, default=None)
```

Comment:
@saleem-latif suggestion: it might make sense to link to IdP via its slug - it should be more human-friendly (especially if `import SAMLProviderConfig` fails).